shopfloor_mobile/static/wms/src/scenario/zone_picking.js

```diff
@@ -16,6 +16,7 @@ const template_mobile = `
             v-if="state.on_scan"
             v-on:found="on_scan"
             :input_placeholder="search_input_placeholder"
+            :autofocus="!screen_info.user_popup"
```

Comment:
Should it not be added to all the scenario ?

Reply:
Good point.

Reply:
BTW any chance to link avoid having to specify this for all scenario in the future? Eg: event or ref?

Reply:
I only implemented it in the scenarios that use this popup (to avoid repeating it in all scenarios).

Regarding the event, I didn't want to touch the base `searchbar` component and keep its logic as is. I prefer just passing this dynamic autofocus, as we don't really gain anything by using events right?

Reply:
We do in fact. Consumers of the searchbar won't have to know about internal implementations.
Anyway, it's ok. Leave it like this for now but add a TODO if you don't mind pls.